client/task-list/tasks.js

```diff
@@ -264,3 +264,15 @@ export function getAllTasks( {
 		'extension'
 	);
 }
+
+export function taskSort( a, b ) {
+	if ( a.completed || b.completed ) {
+		return a.completed ? 1 : -1;
+	}
+	const aLevel = a.level || 'l1';
+	const bLevel = b.level || 'l1';
+	if ( aLevel === bLevel ) {
+		return 0;
+	}
+	return aLevel > bLevel ? -1 : 1;
```

Comment:
This comparison only works because we're expecting single-digit levels. It would break on comparing `l2` to `l11`.

If we remove the `l` as I mentioned about though, we're good here.

Reply:
Changed to an integer, so this should be fine now.